src/api.js

```diff
@@ -2,32 +2,17 @@ import axios from "axios";
 import { toast } from "react-toastify";
 
 export const getConferenceList = async () => {
-  try {
-    const response = await axios.get("http://localhost:8080/conferences");
-    return response.data;
-  } catch (error) {
-    console.log(error);
-    throw error;
-  }
+  const response = await axios.get("http://localhost:8080/conferences");
+  return response.data;
 };
 
 export const getConferenceById = async id => {
-  try {
-    const response = await axios.get(`http://localhost:8080/conferences/${id}`);
-    return response.data;
-  } catch (error) {
-    console.log(error);
-    throw error;
-  }
+  const response = await axios.get(`http://localhost:8080/conferences/${id}`);
+  return response.data;
 };
 
 export const createNewConference = async conference => {
-  try {
-    await axios.post("http://localhost:8080/conferences", conference);
-  } catch (error) {
-    console.log(error);
-    throw error;
-  }
+  await axios.post("http://localhost:8080/conferences", conference);
```

Comment:
Does this POST not return anything?

Reply:
How will you know the call failed now?

Reply:
It does but we don't use it for anything at them moment so that's why we deleted the return statement. For the sake of reducing future changes etc. would it be better to have the same `try{...} catch(error) {throw error}` sequence in each method and then do any logic like `toast` where these methods are called in the rest of the code?